aptos-move/framework/supra-framework/sources/supra_governance.move

```diff
@@ -217,6 +290,29 @@ module supra_framework::supra_governance {
         })
     }
 
+    fun supra_initialize(
+        supra_framework: &signer,
+        voting_duration_secs: u64,
+        min_voting_threshold: u64,
+        voters: vector<address>,
+    ) {
+        multisig_voting::register<GovernanceProposal>(supra_framework);
+
+        move_to(supra_framework, SupraGovernanceConfig {
+            voting_duration_secs,
+            min_voting_threshold,
+            voters,
+        });
+        move_to(supra_framework, SupraGovernanceEvents {
+            create_proposal_events: account::new_event_handle<SupraCreateProposalEvent>(supra_framework),
+            update_config_events: account::new_event_handle<SupraUpdateConfigEvent>(supra_framework),
+            vote_events: account::new_event_handle<SupraVoteEvent>(supra_framework),
+        });
+        move_to(supra_framework, SupraApprovedExecutionHashes {
```

Comment:
Again, just use original `ApprovedExecutionHashes` and not create another `struct` otherwise this would result in Rust layer panic. Rust layer depends on this particular `struct` with this particular name.